modules/mfa/public/send-manager-mfa.php

```diff
@@ -24,7 +24,11 @@
 $logger = LoggerFactory::getAccordingToState($state);
 
 if (filter_has_var(INPUT_POST, 'send')) {
-    $errorMessage = Mfa::sendManagerCode($state, $logger);
+    try {
+        Mfa::sendManagerCode($state, $logger);
+    } catch (Exception $e) {
+        $errorMessage = $e->getMessage();
```

Comment:
Using the ?? null on 44 is laziness. You could define errorMessage on 25 as null. I get the saving vertical space and one instruction slower issue, but explicitly defining would be better in my opinion.

Reply:
Agreed. Thanks for the tip.